ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.ts

```diff
@@ -51,7 +51,7 @@ export class AnalysisContextFormComponent extends FormComponent
     private transformationPaths: MigrationPath[] = [
         {
             "id": 101,
-            "name": "Migration to Red Hat JBoss EAP 7",
+            "name": "Migration to JBoss EAP 7",
```

Comment:
this should be fixed in migration path definition too https://github.com/windup/windup-web/blob/master/services/src/main/resources/migration-paths/migration-paths.xml